tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetCoreBare.cs

```diff
@@ -0,0 +1,39 @@
+// <copyright file="AspNetCoreBare.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETCOREAPP3_0_OR_GREATER
```

Comment:
Why not earlier out of interest?

Reply:
The was my original intent, but it seems it's quite difficult to create a `Start.cs` that's compatible 2.1 / 3.0. It seems the tracer integration tests are separated for this reason (app sec just copied what the tracer did). Doesn't feel like it's worth adding another project solution to test this corner case.

Reply:
Maybe later ...

Reply:
Yeah, that's fine, just checking. You need to if-def `Startup` really, so no worries